explorer/client/src/components/Table.tsx

```diff
@@ -71,34 +77,48 @@ interface IProps extends WithStyles<typeof styles> {
   emptyMsg?: string
 }
 
+const renderCells = ({ r, idx }: { r: any[]; idx: number; }) => (
+  <TableRow key={idx}>
+    {r.map((col: Column, idx: number) => (
```

Comment:
How do you feel about inlining this without `()`?

```jsx
{r.map((col: Column, idx: number) => <TableCell key={idx} column={col} />)}
```

Reply:
I'd like it more inlined too, but linting forces the format.